src/textual/layout.py

```diff
@@ -425,4 +426,5 @@ def update_widget(self, console: Console, widget: Widget) -> LayoutUpdate | None
         update_region = region.intersection(clip)
         update_lines = self.render(console, crop=update_region).lines
         update = LayoutUpdate(update_lines, update_region)
+        log(update)
```

Comment:
Shows the regions that are updating in the logs. Will likely remove in the future, but useful for now.